contrib/brip/__init__.py

```diff
@@ -0,0 +1,24 @@
+from ._brip import *
+
+
+def truncate_nitf_image(img, is_byte, is_scale):
```

Comment:
Does the user use this? Do _they_ have to know if it is byte or not before calling this?

Reply:
Yes, and yes. This can be changed in the future once we have native vil load working.